<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Autenticação</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #add8e6;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelNickname">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #333; font-size: 14px; font-weight: bold; margin-bottom: 5px;</string>
    </property>
    <property name="text">
     <string>Nickname:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUsername">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #333; font-size: 14px; font-weight: bold; margin-bottom: 5px;</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSenha">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #333; font-size: 14px; font-weight: bold; margin-bottom: 5px;</string>
    </property>
    <property name="text">
     <string>Senha:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogin">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #0074d9; font-size: 24px; font-weight: bold; margin-bottom: 5px;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>340</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #333; font-size: 14px; font-weight: bold; margin-bottom: 5px;</string>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLogin">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    background-color: #0074D9;
    color: #FFFFFF; 
    border: 2px solid #0056b3; 
    border-radius: 5px; 
    padding: 5px 15px;
    font-size: 14px;
    font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNickname">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;
border: 1px soolid #ccc;
padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUsername">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;
border: 1px soolid #ccc;
padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSenha">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;
border: 1px soolid #ccc;
padding: 5px;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStatus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;
border: 1px soolid #ccc;
padding: 5px;</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelChatWF">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #0074d9; font-size: 32px; font-weight: bold; margin-bottom: 5px;</string>
    </property>
    <property name="text">
     <string>ChatWF</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelServidor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #333; font-size: 14px; font-weight: bold; margin-bottom: 5px;</string>
    </property>
    <property name="text">
     <string>Servidor:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditServidor">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;
border: 1px soolid #ccc;
padding: 5px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
